angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.stories.ts

```diff
@@ -15,7 +15,33 @@ export default {
     )
   ],
   parameters: {
-    info: markdownDocumentationLinkBuilder('icon'),
+    info: `
+${markdownDocumentationLinkBuilder('icon')}
+
+The icons in Spark are the property of Quicken
+Loans and are not provided directly in our packages. You
+will need to include the Icon yourself. Visit our
+guide on Installing Spark Icons.
```

Comment:
Do we have this guide available yet? If its not going to be available for launch we should hold off on mentioning it. If it is available, we should link it up here.

Reply:
https://github.com/sparkdesignsystem/spark-design-system/issues/2588